src/components/pick-list-item/pick-list-item.e2e.ts

```diff
@@ -127,4 +127,30 @@ describe("calcite-pick-list-item", () => {
 
     expect(removeEventSpy).toHaveReceivedEventTimes(1);
   });
+
+  it("slot keyboard navigation", async () => {
+    const page = await newE2EPage({
```

Comment:
For test stability, can you add a `page.waitForChanges()` after creating the page? Either that or use the following:

```ts
const page = await newE2EPage();
await page.setContent(/* ... */);`
```